packages/management-api/src/managed-pod.ts

```diff
@@ -108,6 +121,36 @@ export class ManagedPod {
     return this._management
   }
 
+  getManagementError(): Error | null {
+    if (!this._management.status.error) return null
+
+    return new Error(`(${this._management.status.error.code}): ${this._management.status.error.message}`)
+  }
+
+  private setManagementError(errorCode: number, errorMsg: string) {
+    this._management.status.error = {
+      code: errorCode,
+      message: errorMsg,
+    }
+  }
+
+  get errorPolling(): ErrorPolling {
+    return this._errorPolling
+  }
```

Comment:
When we only intend to support getter, using JS getter/setter notation is a bit too much without adding any values. Later we might want to make it promise but then it can become a bit painful to rewrite it if it uses the getter/setter notation.

My experiences with Hawtio development now tells me that we should avoid the JS notation altogether.

Reply:
So would you prefer the more java-like style, eg. `getErrorPolling()`, or just making the vars 'public' (well not private - JS scoping tends to be sugar coating at times)?

Certainly, there have been occasions when I prefer creating setters (& rarely getters) when some extra operation is required, eg. the incrementPollCounter(), so I presume that kind of style is ok?

Reply:
Good point. For now, we can choose the explicit getter method: `getErrorPolling()`. I personally see little value in using the Ruby-like implicit getter/setter notation in JS.

And if you need a setter, the style you mention is totally justifiable.